pkg/util/iptables/iptables.go

```diff
@@ -424,7 +426,7 @@ func (runner *runner) restoreInternal(args []string, data []byte, flush FlushFla
 	cmd.SetStdin(bytes.NewBuffer(data))
 	b, err := cmd.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("%v (%s)", err, b)
+		return errors.New(string(b))
```

Comment:
I think that the convention now is to wrap errors https://go.dev/blog/go1.13-errors
```suggestion
		fmt.Errorf("%s: %w", b, err)
```